app/views/user_mailer/account_reset_granted.html.erb

```diff
@@ -47,7 +47,7 @@
 </table>
 
 <p>
-  <%= t('user_mailer.email_confirmation_instructions.footer', confirmation_period: '24 hours') %>
```

Comment:
One other nice effect of this that occurs to me is that this would have been untranslated content, which is now fixed.

You can see "24 hours" in the Spanish version of the [current template](https://idp.dev.identitysandbox.gov/rails/mailers/user_mailer/account_reset_granted.html?locale=es), for example.